tests/testUtil.py

```diff
@@ -55,10 +60,16 @@ def countCDXJEntries(cdxjData):
 
 def startReplay(warcFilename):
     global p
-    pathOfWARC = os.path.join(os.path.dirname(__file__) +
-                              '/../samples/warcs/' + warcFilename)
-    tempFilePath = tempfile.gettempdir() + '/' + ''.join(random.sample(
-        string.ascii_uppercase + string.digits * 6, 12)) + '.cdxj'
+    pathOfWARC = os.path.join(
+        str(Path(os.path.dirname(__file__)).parents[0]) +
+        os.path.sep.join(['', 'samples', 'warcs', warcFilename]))
```

Comment:
```suggestion
    pathOfWARC = os.path.join(
        Path(os.path.dirname(__file__)).parent,
        'samples', 'warcs', warcFilename))
```

Reply:
Much cleaner! I was running into an issue of concatenating POSIXPaths with strings and manually verified that your solution works (mostly to convince myself). Thanks, @ibnesayeed!